tailwind.config.js

```diff
@@ -5,8 +5,9 @@ module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
     screens: {
```

Comment:
What does this do exactly?  Not familiar with the resolutions

Reply:
This is tailwind set sizing requirements for various screen sizes.